src/components/GrowerDetail.js

```diff
@@ -19,6 +19,8 @@ import { GrowerContext } from '../context/GrowerContext';
 import EditGrower from './EditGrower';
 import OptimizedImage from './OptimizedImage';
 import LinkToWebmap from './common/LinkToWebmap';
+import { CopyButton } from './CaptureDetailDialog';
```

Comment:
`CopyButton` needs to be moved to the common components area rather than shared across components like this.